### PR TITLE
Website modifications - adding Blog section

### PR DESCRIPTION
**Pull request by ksunisth**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,9 +10,15 @@
 
 # production
 /build
+server.js
+firebase.js
```
Comment (meganyin13) [inline .gitignore:14]: why are these ignored?

File: src/Components/Landing.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <div>
+      <h1 className={styles.Title}>
+        #SpreadThe
+        <span className={styles.Love}>Love</span>
+      </h1>
+      <p className={styles.SubText}>
+        Week of June 20, 2020
+      </p>
+      <p className={styles.SubText}>
+        Happy mid-June. Whether you're stuck at home or your town is reopening, here are some
```
Comment (meganyin13) [inline src/Components/Landing.tsx:17]: linter error--apostrophe needs to be escaped

File: src/css/SignUp.module.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  font-size: 30px;
+}
+
+.PromptText {
+  text-align: center;
+  font-size: 30px;
+}
+
+.PromptBox {
+  /* display: flex; */
```
Comment (meganyin13) [inline src/css/SignUp.module.css:28]: remove commented code

File: src/firebase/api.ts [hunk 4]
```diff
@@ -16,6 +16,7 @@ firebase.initializeApp(firebaseConfig);
 
 const emailsIDCollection = firebase.firestore().collection('emails_id');
 
-const addEmail = (email: string): Promise<void> => emailsIDCollection.doc(email).set({ email });
+export const addEmail = (email: string):
+Promise<void> => emailsIDCollection.doc(email).set({ email });
 
-export default addEmail;
+export default firebase;
```
Comment (meganyin13) [inline src/firebase/api.ts:22]: I thought we said we were removing `export firebase`?

File: src/Components/Landing.tsx (hunk 2 above)
Comment (ksunisth) [reply src/Components/Landing.tsx]: Fixed this

File: src/css/SignUp.module.css (hunk 3 above)
Comment (ksunisth) [reply src/css/SignUp.module.css]: Fixed this

**Comment by ksunisth** [comment]
> Also, please remove `package-lock.json` we use yarn in this project and having both `package-lock` and `yarn.lock` creates conflicts. Learn more about `yarn` here: https://classic.yarnpkg.com/en/docs/usage

Removed 'package-lock.json'

**Comment by ksunisth** [comment]
> As always, lint checks must pass. Check the GitHub Actions checks to see the errors. I believe most are on escaping apostrophes or missing return type on components (return type should be `React.ReactElement`)

Fixed all the linter issues

**Pull request merged**